src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Completion/DirectiveCompletionItemProvider.cs

```diff
@@ -154,9 +163,11 @@ internal static ImmutableArray<RazorCompletionItem> GetDirectiveCompletionItems(
             if (s_singleLineDirectiveSnippets.TryGetValue(directive.Directive, out var snippetTexts))
             {
                 var snippetCompletionItem = new RazorCompletionItem(
-                    $"{completionDisplayText} ...",
+                    $"{completionDisplayText} {SR.Directive}",
```

Comment:
Should we keep the "..." to indicate its a snippet? Do the other snippets (like `if`, `for`, etc.) do that?

Reply:
IMO, the "..." seems like a good thing to keep.

Reply:
None of the C# snippets (or snippets in any other language) do that AFAIK. I am open to keeping it (other than - is the label getting too long/verbose then?). If we want to keep it (regardless of what other languages do), happy to do that.

Would our re-writer make C# snippet "using statement ..." then instead of the current "using statement"?

Reply:
Also there is a snippet glyph in front of the text indicating it's a snippet that users should be familiar with, as well as a tooltip :) Still happy to keep the "..." if we like it better regardless.

Reply:
Re-added per sync-up discussion